devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -24,8 +24,8 @@ metadata {
 
 		command "reset"
 
-		fingerprint type:"1001", mfr:"0086", prod:"0003", model:"0084", deviceJoinName: "Aeotec Nano Switch 1"
-		fingerprint type:"1001", mfr:"0086", prod:"0103", model:"0084", deviceJoinname: "Aoetec Nano Switch 1"
+		fingerprint mfr:"0086", prod:"0003", model:"0084", deviceJoinName: "Aeotec Nano Switch 1"
+		fingerprint mfr:"0086", prod:"0103", model:"0084", deviceJoinName: "Aoetec Nano Switch 1"
```

Comment:
I meant in the word "Aoetec". Should be "Aeotec".

Reply:
well, it solved my problem with wrong device name, so I thought that I'm done here, sorry